docs/download_options.md

```diff
@@ -50,3 +50,10 @@ In addition, merged objects are not available for all samples or projects, {ref}
 
 Note that even when {ref}`downloading data for all single-cell and single-nuclei samples on the Portal<download_files:Portal-wide downloads>`, merged objects will still be provided per-project.
 There will not be a merged object with all samples from all projects, but a single merged object for each project.
+
+## Multiplexed objects
+
+When downloading a project that contains multiplexed samples, you will have the option to exclude multiplexed samples from the download.
+If selected, the download will contain expression data for only non-multiplexed libraries.
```

Comment:
```suggestion
If selected, the download will contain expression data for only non-multiplexed samples.
```
We should try and be as consistent as possible with using sample or library. Since we are mostly using sample here (and that's the header of the FAQ) let's use that.